vessel/shiplist.csv

```diff
@@ -1,61 +1,62 @@
+FK,R/V,Falkor,CIFR,SOI,ZCYL5,319005600,7928677,82.9,10.2,4.8,,Y,Y
+WS,R/V,F.G. Walton Smith,33WA,Miami,WCZ6292,367020910,8964501,29.3,6.2,2,,Y,Y
+FL,R/V,FLIP,31FP,SIO,WI7115,338040561,,108.2,5.1,91.4,,Y,Y
+HLY,R/V,Healy,33HQ,USCG,NEPP,303902000,9083380,128,8.8,11,WAGB-20,Y,Y
+HRS,R/V,Hugh R. Sharp,33H5,UDel,WDC6908,367063060,9388443,44.5,6.2,2.7,,Y,Y
+JCR,RRS,James Clark Ross,,BAS,,,,,,,,Y,N
+TY,CCGS,John P. Tully,,CCGS,,,,,,,,Y,N
+KOK,R/V,Ka`imikai-O-Kanaloa,33KI,Hawaii,WBN4310,303165000,7806233,68,5.7,4.2,,Y,Y
+KA,F/V,Karoline Marie,,,,,,,,,,Y,N
+KM,R/V,Kilo Moana,33KB,Hawaii,WDA7827,369565000,9229037,56.7,7.7,7.6,AGOR-26,Y,Y
+KN,R/V,Knorr,316N,WHOI,KCEJ,367246000,7738618,85,7.2,6,AGOR-15,(N),Y
+LMG,R/V,Laurence M. Gould,33LG,USAP,WCX7445,368138000,9137337,70.1,5.8,5.7,,Y,Y
+MGL,R/V,Marcus G. Langseth,33H3,LDEO,WDC6698,367059880,9010137,71.6,6.7,6.7,,Y,Y
+MM,R/V,Maria S. Merian,,IOW,,�211753000,9274197,,,,,Y,N
+EW,R/V,Maurice Ewing,3230,LDEO,WLDZ,366167000,8203153,70.1,6.7,5.3,,(N),Y
+MV,R/V,Melville,318M,SIO,WECB,366784000,7738591,85,7.2,5,AGOR-14,(N),Y
+MW,R/V,Moana Wave,32MW,Hawaii,WUS9293,367627000,7319008,50.3,6.7,3.5,AGOR-22,(N),Y
+NBP,R/V,Nathaniel B. Palmer,3206,USAP,WBP3210,366610000,9007257,93.9,8.5,7.2,,Y,Y
+AR,R/V,Neil Armstrong,33VB,WHOI,WARL,338767000,9688946,72.5,7,4.6,AGOR-27,Y,Y
+NH,R/V,New Horizon,32NM,SIO,WKWB,303273000,7723821,51.8,6.3,3.5,,(N),Y
+OC,R/V,Oceanus,32OC,OSU,WXAQ,367977000,7603617,53.9,6.2,6,,Y,Y
+EX,R/V,Okeanos Explorer,334A,OMAO,WTDH,369888000,8835114,68.3,6.2,5.2,,Y,Y
+PE,R/V,Pelican,32PE,LUMCON,WSK3051,367652000,8517009,35.4,4.7,2.9,,Y,Y
+PS,R/V,Point Sur,32P0,LUMCON,WSC2276,366974890,8023539,41.1,5.7,2.7,,Y,Y
+PSEA,R/V,Polar Sea,32L9,USCG,NRUO,367878000,7391252,121.6,9,10,WAGB-11,(N),Y
+PSTAR,R/V,Polar Star,32PZ,USCG,NBTM,303991000,7367471,121.6,9,10,WAGB-10,(N),Y
+S,R/V,Robert C. Seamans,33GY,SEA,WDA4486,369503000,9245483,40.8,5.1,4,,Y,Y
+RC,R/V,Robert D. Conrad,31RD,LDEO,WHBA,,7742140,63.7,13.8,4.9,AGOR-3,(N),Y
+SP,R/V,Robert Gordon Sproul,32QU,SIO,WSQ2674,303272000,8128747,38.1,5.4,2.5,,Y,Y
+RR,R/V,Roger Revelle,33RR,SIO,KAOU,367800100,9075228,84.4,7.7,5,AGOR-24,Y,Y
+RB,R/V,Ronald H. Brown,33RO,OMAO,WTEC,368996000,9105786,83.5,7.7,5.2,,Y,Y
+SR,R/V,Sally Ride,,SIO,WSAF,369140000,9695171,72.5,7,4.6,AGOR-28,Y,Y
+SAV,R/V,Savannah,33H6,SkIO,WDA5407,366819770,8992651,28,5.1,2.4,,Y,Y
+SI,M/V,Scartlett Isabella,,BHC,,,,,,,,Y,N
+STD,R/V,SeaTow Delmarva,,SeaTow,,,,,,,,Y,N
+STO,R/V,SeaTow Obsession,,SeaTow,,,,,,,,Y,N
+SKQ,R/V,Sikuliaq,33BI,UAF,WDG7520,338417000,9578945,80,7.3,5.7,,Y,Y
+TS,F/V,Tenacious II,,TM,,,,,,,,Y,N
+TN,R/V,Thomas G. Thompson,3250,UW,KTDQ,366345000,8814419,83.5,7.5,5.9,AGOR-23,Y,Y
+WT,R/V,Thomas Washington,31WT,SIO,KGWU,,,63.7,13.8,4.9,AGOR-10,(N),Y
+TA,R/V,Tioga,,WHOI,,,,,,,,Y,N
+V,R/V,Vema,31VE,LDEO,J8RU,,7738383,,,,,(N),Y
+WJ,M/V,Warren Jr.,,BHC,,,,,,,,Y,N
+WOO,F/V,Wahoo,,Meredith,,,,,,,,Y,N
+W,R/V,Wecoma,32WC,OSU,WSD7079,303014000,7604439,56.4,7.2,6,,(N),Y
```

Comment:
Reviewed by RT - addition of M/V Blackhawk confirmed.

Reply:
Overall looks okay.  Some weird things going on with ".0"  on numbers – looks like they were added at some point to the IMO#s (probably by Excel), and now they are being removed.